### PR TITLE
fix IF EXISTS for create alias and ALTER ALIAS

### PR DESCRIPTION
**Pull request by HannesSandberg**
https://github.com/neo4j/neo4j-documentation/issues/1518

### CODE REVIEW

**Review by martin-neotech — APPROVED** [review]
Super!

**Review by nadja-muller — APPROVED** [review]

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 1]
```diff
@@ -383,7 +383,7 @@ label:functionality[]
 label:new[]
 [source, cypher, role="noheader"]
 ----
-CREATE ALIAS ... [IF EXISTS]
+CREATE [OR REPLACE] ALIAS ... [IF NOT EXISTS]
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]:
```suggestion
CREATE ALIAS ... [IF NOT EXISTS]
```

And add a new row with the

```
CREATE OR REPLACE ALIAS ...
```

**Review by martin-neotech — CHANGES_REQUESTED** [review]
It is better to create a new row with the `CREATE OR REPLACE` command because it is basically another command and it does not allow the `IF NOT EXISTS` either.

**Review by martin-neotech — APPROVED** [review]
Super! Thanks

**Pull request merged**